index.bs

```diff
@@ -163,6 +163,38 @@ spec: RFC8941; urlPrefix: https://datatracker.ietf.org/doc/html/rfc8941#
     to use the Geolocation API, but any other ports on "<code>https://example.com</code>"
     could use it too.</p>
   </div>
+  <div class="example">
+    <p>JSPlaygroundCorp Inc. wants to host user-generated web applications, but wants the
+    browser to manage their permissions to use powerful features in isolation of each other.
+    This can be accomplished by creating discrete subdomains for each piece of web-content
+    or web-content creator, and navigating them as top-level documents (framework and
+    user-content can still be separated using same-origin iframes).
+
+    JSPlaygroundCorp should avoid iframing the web-content using the "<code>allow</code>"
+    attribute from its own domain.
+  </div>
+  <div class="example">
+    <p>PlatformCorp Inc. wants to offer a marketplace of embeddable third-party components
+    to build from or games to play. It wants to delegate
+    the use of powerful features like the getUserMedia API responsibly. It keeps
+    track of which components need a feature, using bespoke "install" UX to keep end-users
+    in charge.</p>
+    <p>Camera and microphone are disabled by default in all cross-origin frames.
+    Each third-party component has a subdomain, and can be embedded in a
+    cross-origin iframe. PlatformCorp can use the "<code>allow</code>" attribute on
+    the iframe element to control whether to delegate camera or microphone access or
+    not to each subdomain.
+    An iframe where "plugin1" and "plugin3" should have camera access and "plugin2"
+    should have microphone access might look like this:
+    <pre>
+    &lt;iframe
+     <a href="#iframe-allow-attribute">allow</a>="camera //plugin1.site.com //plugin3.site.com; microphone //plugin2.site.com"
```

Comment:
Bikeshed supports syntax for linking correctly to an attribute of an element. Might be good to link to HTML spec here.